app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
Phân biệt `find` `find_by` giúp chị

ở đây dùng find_by, bỏ ngoặc, và thêm code xử lý khi bị nil

Reply:
Em trả lời. 
Method find nếu không tìm thấy record nào trong bảng users sẽ bắn ra exception
Còn method find_by nếu không tìm thấy record nào trong bảng users thì return về nil.

Reply:
thêm nữa, nếu tìm thấy data thì object trả của 2 method này có j khác nhau

Reply:
nếu tìm thấy thì object trả về hoàn toàn giống nhau ạ.